pom.xml

```diff
@@ -469,6 +480,31 @@
           <useSystemClassLoader>false</useSystemClassLoader>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>2.0.0</version>
+        <executions>
+          <execution>
+            <configuration>
+              <verbose>false</verbose>
+              <thirdPartyFilename>THIRD-PARTY-Licenses.txt</thirdPartyFilename>
+              <useMissingFile>true</useMissingFile>
+              <missingFile>THIRD-PARTY-Missed-Licenses.txt</missingFile>
+              <deployMissingFile>true</deployMissingFile>
```

Comment:
Can you add `<failOnMissing>true</failOnMissing>` and add the missing licenses?

Reply:
If filled in the missing licenses and added <failOnMissing>true</failOnMissing>.